gcp/appengine/frontend_handlers.py

```diff
@@ -403,6 +403,7 @@ def group_versions(versions):
 def markdown(text):
   """Render markdown."""
   if text:
+    text = text.replace('<', '&lt;').replace('>', '&gt;')
     return markdown2.markdown(text, extras=['fenced-code-blocks'])
```

Comment:
Rather than manually replacing the tags, use markdown2's safe_mode option:
```
markdown2.markdown(text, safe_mode = 'escape', extras=...)
```

Reply:
I can't really find any documentation for it, but here it is in the source code:
https://github.com/trentm/python-markdown2/blob/master/lib/markdown2.py#L1335

Reply:
+1, this seems preferable and slightly more comprehensive, looking at the code. Good find @another-rex

Reply:
Thanks, couldn't find any documentation on this before.

I had avoided things like html.escape before because I wanted to preserve things like `&amp;` wrt rendering (and not getting that double escaped), but maybe that's also unsafe in some way I haven't considered.

Reply:
Done!

Reply:
It seems like it's safe for `&` to not be escaped as long as we are not putting it in any javascript lines, which we are not. Are there examples of the potential double escaping?

Reply:
No examples I can find. We can revisit this if this ever comes up, but either way it's not a huge deal.